src/main/java/com/example/springboot/entity/Book.java

```diff
@@ -0,0 +1,68 @@
+package com.example.springboot.entity;
+
+/**
+ * This is a basic class for book object.
+ * @author Kohut Dmytro
+ * @version 1.0
+ */
+public class Book {
+	
+	private Integer id;
+	private String title;
+	private String author;
+	private Integer readerId;
```

Comment:
so one book can read only one reader? Book like a hours ? and if two users takes the same book during month we will rewrite this information, yep? Why we need history, no no no. It is not interesting for business. Never-ever.